.makefile/daq-kube.mk

```diff
@@ -1,3 +1,3 @@
 ${MAKEFILE_DIR}/daq-kube/.kluctl.yaml:
 	@echo "Fetching daq-kube repo..."
-	@cd ${MAKEFILE_DIR} && git clone https://github.com/DUNE-DAQ/daq-kube.git --recursive
+	@cd ${MAKEFILE_DIR} && git clone  https://github.com/DUNE-DAQ/daq-kube.git --recursive
```

Comment:
I'm not sure this extra space adds anything....

Reply:
The space was a mistake after changing back from git@github... Back to https... I have now removed the space